public/modules/articles/articles.viz.js

```diff
@@ -499,6 +515,9 @@ var valign, stacked, articlesScope, aligned = true;
                                         .popover(function () {
                                             return section_hover(d, curS)
                                         })
+                                        .on("mouseenter", function() {
+                                            $(".popover").addClass("articles-popover");
+                                        })
                                         .text(clean_premier(titre_section(sub_section(curS), longlabel)));
                                     ct++;
```

Comment:
Okay, so I found a solution and that was not easy. Bootstrap enables passing a template for the popover so we just need to pass a template with a specific class. Except it doesn't work, because we actually use d3-bootstrap-plugins, and it does *only* implement *basic* bootstrap popover/tooltip features.

So I rewrote part of d3-bootstrap-plugins in order to be able to pass additional classes. This is definitively code to get rid of ASAP :)